File: 88-MergeSortedArray/88-MergeSortedArray.cpp
// Last updated: 7/16/2026, 4:38:52 PM
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        for(int i = 0; i<n; i++){
            nums1[m+i]=nums2[i];
        }
        sort(begin(nums1), end(nums1));
        for(int x : nums1) cout<<x<<" ";
    }
};

File: 88-MergeSortedArray/88-MergeSortedArray.py
# Last updated: 8/3/2026, 8:27:13 PM
1class Solution:
2    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
3        for i in range (m,m+n):
4            nums1[i]=nums2[i-m]
5        nums1.sort()
6        